### PR TITLE
🚨 Fix warnings

### PR DESCRIPTION
**Pull request by warrenseine**

### CODE REVIEW

**Review by DannyvanderJagt — CHANGES_REQUESTED** [review]

File: src/components/Confirm/Confirm.tsx [hunk 1]
```diff
@@ -76,7 +77,7 @@ export const Confirm = (props: ConfirmProps) => {
       document_type: props.documentType,
       country_code: props.idDocumentIssuingCountry?.country_alpha2,
     })
-  }, [])
+  }, [props])
```
[the remark this answers is not in the document]
Comment (DannyvanderJagt) [reply src/components/Confirm/Confirm.tsx:80]: The props shouldn't change when the screen already has been loaded.

[the remark this answers is not in the document]
Comment (warrenseine) [reply src/components/Confirm/Confirm.tsx:80]: Agreed, so there's no problem adding it in the dependency array.

[the remark this answers is not in the document]
Comment (DannyvanderJagt) [reply src/components/Confirm/Confirm.tsx:80]: In case they do, we now trigger a new analytics event which we shouldn't.

**Review by DannyvanderJagt — APPROVED** [review]
Thanks for changing 👍

**Pull request merged**